### ISSUE TITLE
The method 'ConnectionEntry' isn't defined

### ISSUE THREAD

**Issue by definitelyme** [Iteo/connecteo#33]
```bash
The method 'ConnectionEntry' isn't defined for the type 'Modules'.
Try correcting the name to the name of an existing method, or defining a method named 'ConnectionEntry'.
```

Did you export this class?

**Comment by definitelyme** [comment]
See this

![Screenshot 2023-10-20 at 00 39 00](https://github.com/Iteo/connecteo/assets/28307747/2ab565dc-920c-4c43-ac10-ab6271c97600)

**Comment by Devi88** [comment]
Hello, thanks for issue. @definitelyme 
Yes you are right, I forgott export a connection_entry class
Fix is added to version 2.1.2

**Comment by Devi88** [comment]
Hello, thanks for issue. @definitelyme
Yes you are right, I forgott export a connection_entry class
Fix is added to version 2.1.2
